encodercontrol.h

```diff
@@ -18,12 +18,14 @@ class EncoderControl : public QObject
     Port            *port;
     QString         portName;
     uint8_t         word_cnt;
+    // QA: where do you use this variable?
     int             zero_mark_number;
     float           position_mm;
     float           final_position_mm;
     int             prev_position_raw;
     int             position_raw;
     int             delta;
```

Comment:
Look at the [void EncoderControl::analyzePosition( QString str )
](https://github.com/Muzalevsky/SITH/blob/f414b45154c37f7fa9f14b7438103b53c70be056/encodercontrol.cpp#L77)

Reply:
Ops! My fault